share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java

```diff
@@ -25,139 +25,109 @@
  */
 package org.alfresco.web.config.properties;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-
 import org.alfresco.web.config.util.BaseTest;
-import org.alfresco.web.site.servlet.config.AIMSConfigElement;
-import org.junit.Test;
-import org.keycloak.representations.adapters.config.AdapterConfig;
+import org.alfresco.web.site.servlet.config.AIMSConfig;
+import org.junit.Assert;
+import org.junit.Before;
 import org.springframework.core.io.Resource;
 import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
 import org.springframework.extensions.config.Config;
-import org.springframework.extensions.config.ConfigElement;
 import org.springframework.extensions.config.xml.XMLConfigService;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class AIMSConfigTest extends BaseTest
 {
-
-    public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
+    public static final String CLASSPATH_SHARE_CONFIG_PROPERTIES = "classpath*:alfresco/module/*/share-config.properties";
     private static final String TEST_CONFIG_AIMS_BASIC_XML = "test-config-aims-basic.xml";
-    protected XMLConfigService configService;
 
-    protected List<String> getConfigFiles()
+    @Before public void setUp()
```

Comment:
Why do we need this?

Reply:
@cristina-sirbu Removed the unneeded setUp method.